shard.yml

```diff
@@ -31,7 +31,7 @@ dependencies:
     version: ~> 0.4.0
 
   liquid:
-      github: TechMagister/liquid.cr
+    github: TechMagister/liquid.cr
```

Comment:
Nice!

Can you add the version as well?, ( And hope @TechMagister release the right tag soon :sweat_smile: )

```yaml
  liquid:
    github: TechMagister/liquid.cr
    version: ~> 0.2.4
```

Or maybe we should fork it as temporal workaround

```yaml
  liquid:
    github: amberframework/liquid.cr
    version: ~> 0.2.4
```

Reply:
Lets not fork it we have too many forks already. Lets hope he will update the tag soon

Reply:
@eliasjpr Fixed! https://github.com/TechMagister/liquid.cr/issues/29